src/Microsoft.ML.StandardTrainers/Standard/MulticlassClassification/MulticlassNaiveBayesTrainer.cs

```diff
@@ -294,25 +294,25 @@ private NaiveBayesMulticlassModelParameters(IHostEnvironment env, ModelLoadConte
             : base(env, LoaderSignature, ctx)
         {
             // *** Binary format ***
-            // int: _labelCount
-            // int[_labelCount]: _labelHistogram
+            // int: _labelCount (read during reading of _labelHistogram in ReadLongArray())
```

Comment:
I added this comment as we are not explicitly reading _labelCount here, but in `ReadLongArray()` as shown below:

https://github.com/dotnet/machinelearning/blob/214926fcf5753ad62acf32c0759bdaf8fcd13b73/src/Microsoft.ML.Core/Utilities/Stream.cs#L681-L688